video_xblock/mixins.py

```diff
@@ -103,8 +103,13 @@ def convert_caps_to_vtt(caps):
     def vtt_to_text(vtt_content):
         """
         Utility method to extract text from WebVTT format transcript.
+        
+        If the vtt_content parameter comes as bytes, this 
+        parameter must be converted to a string.
```

Comment:
Are you sure it always comes as bytes? If so, why do we need that isinstance check?

Anyway I don't think this should be added to the docstring

Reply:
I'm not sure that only bytes come.